instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/otlp/OtlpSpanExporterAutoConfiguration.java

```diff
@@ -33,26 +36,25 @@
 public class OtlpSpanExporterAutoConfiguration {
 
   @Bean
-  @ConditionalOnMissingBean
-  public OtlpGrpcSpanExporter otelOtlpGrpcSpanExporter(OtlpExporterProperties properties) {
-    OtlpGrpcSpanExporterBuilder builder = OtlpGrpcSpanExporter.builder();
-
-    String endpoint = properties.getTraces().getEndpoint();
-    if (endpoint == null) {
-      endpoint = properties.getEndpoint();
-    }
-    if (endpoint != null) {
-      builder.setEndpoint(endpoint);
-    }
-
-    Duration timeout = properties.getTraces().getTimeout();
-    if (timeout == null) {
-      timeout = properties.getTimeout();
-    }
-    if (timeout != null) {
-      builder.setTimeout(timeout);
-    }
-
-    return builder.build();
+  @ConditionalOnMissingBean({OtlpGrpcSpanExporter.class, OtlpHttpSpanExporter.class})
```

Comment:
What about if the user adds another type of span exporter (for logging by example)?

Reply:
see below